examples/discovery-search-app/src/setupProxy.js

```diff
@@ -8,18 +8,18 @@ const cors = require('cors');
 const bodyParser = require('body-parser');
 
 if (serverEnv.error) {
-  console.warn(serverEnv.error);
+  throw new Error(serverEnv.error);
 }
 
 if (clusterEnv.error) {
-  console.warn(clusterEnv.error);
+  throw new Error(clusterEnv.error);
 }
 
-const RELEASE_PATH = process.env.RELEASE_PATH;
-const BASE_URL = process.env.BASE_URL;
+const RELEASE_PATH = process.env.RELEASE_PATH || '/example/release/path';
+const BASE_URL = process.env.BASE_URL || 'http://example.com';
```

Comment:
@maniax89 what are your thoughts on still giving these default values? If `.env.local` or `.server-env` don't exist, we'll throw exceptions. But that doesn't guarantee that any of the specific env vars exist within those files. Should we add checks for each one, or leave these default values in place?

Reply:
I think it makes sense to check each one if the authenticator doesn't already

Reply:
can we throw an error if `RELEASE_PATH` or `BASE_URL` are undefined along with an action the user to take? (like "please make sure you have defined `BASE_URL` in the `.server-env` file before attempting this command again")

Reply:
I want to say that I tried, but it was causing additional problems, because `setupProxy` is run for both server and client startup (and the env vars are only present in server startup). I removed those changes, since the priority with this issue was to get the setup instructions in a good state. If we hear from customers that that part is unclear, I'm willing to return to this, or I could spin up a smaller issue to deal with it.

Reply:
yes that makes sense. probably some additional checks are needed when starting the client to ensure those variables are present since like you said both server and client rely on the same variables

Reply:
Actually, if we add a cypress env var like you suggested elsewhere, we might be able to bypass that problem. I'm going to fiddle with it and see if that works.